client/src/GlobalStyle.js

```diff
@@ -9,15 +9,20 @@ const GlobalStyle = createGlobalStyle`
 }
 
 html {
+  width: 100%;
+  height: 100%;
   font-family: "Orbitron", Arial, sans-serif;
   background-image: url(${img});
   background-repeat: no-repeat;
-  background-size: 100vmax;
+  background-size: cover;
+  color: #fff;
+  background-position: center;
 }
 
 body {
-  display: flex;
-  justify-content: center;
+  width: 100%;
+  height: 100%;
```

Comment:
Do html and the body need width and height?

Reply:
In this case, yes. Because of the positioning of the background-image. In the body not so much, but our css specialist in class said, it would be better to do so.